### PR TITLE
#5042: Quick fix to update flow creation errors in low priority tests

### PR DESCRIPTION
**Pull request by pkazlenka**

### CODE REVIEW

**Review by niksv — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Review by artHalMirantis — APPROVED** [review]

**Comment by niksv** [comment]
tests are red

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java [hunk 1]
```diff
@@ -142,12 +142,16 @@ public void restartFloodlight(String region) {
 
     @Override
     public List<FloodlightResourceAddress> knockoutSwitch(Switch sw, List<String> regions) {
+        throw new RuntimeException("This method is not allowed to run on hardware environment. " +
```
Comment (dmitrii-beliakov) [inline src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java]: As an option you can throw `UnsupportedOperationException`. From the description I understood that it's ok to use it on some other environments, should there be some if-condition?

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java [hunk 2]
```diff
@@ -142,12 +142,16 @@ public void restartFloodlight(String region) {
...
+                "See https://github.com/telstra/open-kilda/issues/5074 for details");
+        /* Don't delete this code. It's commented out to allow to build the class
```
Comment (dmitrii-beliakov) [inline src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java:147]: The code will be available in VCS history. It's easy to restore deleted code from previous versions, so no need for commenting out unreachable code? Is there some additional value in this comment?

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java (hunk 1 above)
Comment (pkazlenka) [reply src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java]: If I get the code right, dependency injections replaces if's in this case.
Changed the type of exception thrown.

File: src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java (hunk 2 above)
Comment (pkazlenka) [reply src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java]: Fixed.

**Review by dmitrii-beliakov — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Comment by niksv** [comment]
@pkazlenka Please fix conflicts

**Comment by niksv** [comment]
@pkazlenka please squash your commits. You have 5 commits with equal commit messages

**Review by niksv — APPROVED** [review]

**Pull request merged**